content/creator/getting-started/index.mdx

```diff
@@ -0,0 +1,263 @@
+---
```

Comment:
I merged all of the getting started sections into a single article. We can easily break them out again as needed, but my thought was that pieces should be "as makes sense for a single reading" and this feels like a section we'd want people to read all in one go.